src/shared/services/configuration/endpoint-definitions.js

```diff
@@ -89,12 +89,17 @@ export default {
   // sources from which an incident originated. No Swagger documentation
   // available.
   SOURCES_ENDPOINT: '/signals/v1/private/sources/',
+  // Private. Used by the incident management module to POST status messages for a category.
+  STANDARD_TEXTS_CATEGORY_ENDPOINT:
+    '/signals/v1/private/status-messages/category/',
+  // Private. Used by the incident management module to retrieve and query standard texts.
+  STANDARD_TEXTS_ENDPOINT: '/signals/v1/private/status-messages/',
+  // Private. Used by the incident management module to retrieve and search standard texts.
+  STANDARD_TEXTS_SEARCH_ENDPOINT: '/signals/v1/private/status-messages/search',
   // Private. Used by the incident management module to `GET` and `POST`
   // default response texts for incident handling. See
   // https://api.data.amsterdam.nl/api/swagger/?url=/signals/swagger/openapi.yaml#/default/get_signals_v1_public_terms_categories
   // for reference.
```

Comment:
I think the comments are slightly mixed up. The test here does not correspond with the TERMS_ENDPOINT